docs/development/remote.md

```diff
@@ -208,6 +208,21 @@ $ rmate [options] file
 ```
 and the file should open up in VS Code. Any changes you make now will be saved remotely.
 
+##### PyCharm
+
+[PyCharm](https://www.jetbrains.com/pycharm/) is an all-in-one tool for Python development,
+and the Professional version is free to use for students and FOSS contributors.
```

Comment:
Actually I think JetBrains only provides open source licenses to "core team developers" if I am not mistaken https://www.jetbrains.com/community/opensource/

Reply:
Ah alright. I mistook the application terms as license terms and figured that regular committers also get access. 
Sorry, I'll just change that.